src/ResourceManager/LogicApp/Commands.LogicApp.Test/ScenarioTests/IntegrationAccountGeneratedIcnTests.cs

```diff
@@ -37,7 +37,7 @@ public IntegrationAccountGeneratedIcnTests(Xunit.Abstractions.ITestOutputHelper
         /// </summary>
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
-        public void TestGetIntegrationAccountGeneratedIcnWithoutAgreementType()
+        public void TestGetIAGeneratedIcnWithoutAgreementType()
```

Comment:
Do we need to change the recording json file path?

Reply:
For net452, yes. That was changed here: https://github.com/Azure/azure-powershell/pull/7497/files#diff-2ba361f18e7b81bc1187080711233190R186